docs/sources/reference/components/prometheus/prometheus.exporter.ssh.md

```diff
@@ -0,0 +1,205 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/prometheus/prometheus.exporter.ssh/
+aliases:
+  - ../prometheus.exporter.ssh/ # /docs/alloy/latest/reference/components/prometheus.exporter.ssh/
+description: Learn about prometheus.exporter.ssh
+title: prometheus.exporter.ssh
+---
+
+# prometheus.exporter.ssh
+
+The `prometheus.exporter.ssh` component embeds an SSH exporter for collecting metrics from remote servers over SSH and exporting them as Prometheus metrics.
+
+## Usage
+
+```
+prometheus.exporter.ssh "LABEL" {
+  // Configuration options
+}
+```
+
+## Arguments
+
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional unless specified. Omitted fields take their default values.
+
+| Name              | Type     | Description                                        | Default | Required |
+| ----------------- | -------- | -------------------------------------------------- | ------- | -------- |
+| `verbose_logging` | `bool`   | Enable verbose logging for debugging purposes.     | `false` | no       |
```

Comment:
Do we really need such a argument? Normally users would configure the logging level through the [logging block](https://grafana.com/docs/alloy/latest/reference/config-blocks/logging/).